web3/contract.py

```diff
@@ -1132,6 +1133,46 @@ def _parse_logs(self, txn_receipt):
                 continue
             yield decoded_log
 
+    @combomethod
+    def createFilter(
+            self, *,  # PEP 3102
+            argument_filters=dict(),
+            fromBlock=None,
+            toBlock="latest",
+            address=None,
+            topics=list()):
```

Comment:
Same here, should be changed to `None` and then defaulted to an empty list conditionally if it is `None`